homeassistant/components/azure_devops/__init__.py

```diff
@@ -26,9 +26,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         entry=entry,
     )
 
-    # Store the coordinator in hass data
-    hass.data.setdefault(DOMAIN, {})
-    hass.data[DOMAIN][entry.entry_id] = coordinator
+    # Store the coordinator in runtime data
```

Comment:
and type the entry here with the custom one as well (and any other place you have a blank `ConfigEntry`)